### PR TITLE
Slightly improve the README

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

I always wanted to put some more love into the main README file. So, this is just a humble start.

- Adjust some wordings
- Put "Features" in a real document section
- Add the term "document" (store) in order to improve the understanding of what "dynamic schemas" really mean
- Mention support for the PostgreSQL wire protocol and adjust Docker command
- Mention client and shell options how to connect to CrateDB
- Mention the CrateDB Kubernetes Operator
- Add more pointers to specific parts of the documentation within the "Help" section and add pointers to the cloud offering

Let me know what you think about these slight adjustments and also if you would like to see other details to be mentioned here. Imagine people looking at this solely by arriving at the GitHub repository and not investing time reading any out-of-band documentation at all.

With kind regards,
Andreas.

### CODE REVIEW

**Review by seut — APPROVED** [review]
👍 Lgtm

File: README.rst [hunk 1]
```diff
@@ -15,28 +15,46 @@
...
+The smallest CrateDB clusters can easily ingest tens of thousands of records
+per second. The data can be queried, ad-hoc, in parallel across the whole
+cluster in real time.
+
+
+Features
+========
+
+- Standard SQL interface available via HTTP API and PostgreSQL wire protocol.
+- Dynamic schemas, queryable objects, geospatial features, time series data support
```
Comment (mechanomi) [inline README.rst:38]:
```suggestion
- Dynamic schemas, queryable objects, geospatial features, time series data support,
```

oxford comma :)

File: README.rst [hunk 2]
```diff
@@ -60,13 +81,17 @@ contributions!
...
 ====
 
 Looking for more help?
 
-- Read `the project documentation`_
-- Check out our `support channels`_
+- Read `the project documentation`_.
+- Check out our `support channels`_.
+- For running CrateDB in the cloud, you are welcome to visit our offering
+  at `CrateDB Cloud`_ and its documentation at `CrateDB Cloud Tutorials`_.
```
Comment (mechanomi) [inline README.rst:93]: this is a little too wordy for this list, I think. how about something like:

- `CrateDB Cloud` is a fully-managed CrateDB *Database as a Service* (DBaaS) by `Crate.io`_

however, you might wanna check with a PM for phrasing. I expect we have a collection of pre-written sentences like this

File: README.rst [hunk 3]
```diff
@@ -60,13 +81,17 @@ contributions!
...
 
+
 Help
 ====
 
 Looking for more help?
 
-- Read `the project documentation`_
-- Check out our `support channels`_
+- Read `the project documentation`_.
```
Comment (mechanomi) [inline README.rst:90]: I saw a PR recently (though I have lost track of it) that moved the documentation links into the main body of the README (for a different Crate project)

this link links to `/docs/`, which bounces you to the tutorials. at a minimum, we should probably change this to say something like "Try one of our beginner tutorials, how-to guides, or check out the reference manual" (adding links around "beginner tutorials, "how-to guides", and "reference manual")

alternatively, we could spin this out into a short subsection that links to the different docs projects one by one (perhaps using a list format). not sure

**Review by mechanomi — COMMENTED** [review]
excellent stuff. thank you!!

File: README.rst (hunk 1 above)
Comment (amotl) [reply README.rst]: Thanks.

File: README.rst (hunk 2 above)
Comment (amotl) [reply README.rst]: I've reworded the sentence according to your suggestion. Feel free to give advice for further adjustments.

**Comment by amotl** [comment]
Thanks for your suggestions, @mechanomi - I appreciate it! Feel free to get back to me for another iteration based on the current state.

File: README.rst (hunk 3 above)
Comment (mechanomi) [reply README.rst]: ah of course. my bad. sorry! faulty memory hehe. thanks!

**Review by mechanomi — COMMENTED** [review]
thanks for making changes! this looks great! good to go imo. left one comment about a minor RST style issue

File: README.rst [hunk 4]
```diff
@@ -76,3 +105,13 @@ Looking for more help?
 .. _support channels: https://crate.io/support/
 .. _the introductory docs: https://crate.io/docs/crate/tutorials/
 .. _the project documentation: https://crate.io/docs/
+.. _beginner tutorials: https://crate.io/docs/crate/tutorials/
```
[the remark this answers is not in the document]
Comment (amotl) [reply README.rst:108]: All right. I've just pushed another update.

**Review by mechanomi — APPROVED** [review]
